pages/ccf-3d-reference-library.html

```diff
@@ -119,47 +97,51 @@ <h4 class="mt-5">Reference Organs</h4>
                                 <ul class="nav nav-tabs bg-light" id="myTab" role="tablist">
                                     <li class="nav-item">
                                         <a class="nav-link active" id="bone-tab" data-toggle="tab" href="#bone"
-                                            role="tab" aria-controls="home" aria-selected="true">Bone Marrow & Blood/Pelvis</a>
+                                            role="tab" aria-controls="home" aria-selected="true"><img src= "/img/icons/ico-organs-pelvis f.svg" Width=45.7px height=45.7px ><br/>Bone Marrow &<br/> Blood/Pelvis</a>

```

Comment:
Don't use starting / for links. Use '../img/XXXX'. When deployed to production it's under /ccf/ and thus all references to /img will fail.